.changeset/shaggy-camels-remain.md

```diff
@@ -0,0 +1,7 @@
+---
+'@backstage/plugin-auth-backend': patch
```

Comment:
This doesn't seem to have a change below?

Reply:
🤦 I misread, hang on.

Reply:
Fixed it, sorry for the trouble.